_pages/dev/extending-checkout.md

```diff
@@ -403,7 +479,7 @@ With the account payment option, you can only select the addresses of cost cente
 
 ### Known Issues
 
-When you start the checkout process, if the screen is blank and seems to be stuck loading, it means that you are missing the payment method step from your CMS. To resolve this issue, make sure that you are using `spartacussampledata` version 3.0.0 or newer, and that you are using SAP Commerce Cloud 2005 or newer.
+When you start the checkout process, if the screen is blank and seems to be stuck loading, it means that you are missing the payment method step from your CMS. To resolve this issue, make sure that you are using `spartacussampledata` version 5.0.0 or newer, and that you are using SAP Commerce Cloud 2105 or newer.
```

Comment:
Let's add an example of extending the checkout.
Let's say customers wanted to add a new payment method (e.g. pay with cash upon delivery), how would they go about it?

Reply:
Should we really include this? It's more of a specific example because they'd have to change a few things

1. make sure the backend supports cash upon delivery, where the `miscs controller` for the endpoint `GET` /{baseSiteId}/cardtypes includes this type code and name as the dropdown for the type of payment methods just gets looped on from this 'list'
2. Once the backend supports it and exposed through that endpoint, they would need to extend `CheckoutPaymentFormComponent` in order for them to have the payment form data to emit this new type of information to the parent component `CheckoutPaymentMethodComponent` as well as copy pasting the template and hiding / showing the input related to cash upon delivery
2.1 might also need to augment the interface `PaymentDetails` for additional data that it requires to do this payment
2.2 might as well make sure this interface in the backend is supporting any of the new additional property for `users/${userId}/carts/${cartId}/payment/sop/response` 


all in all, we don't have to changed things from other layers as we're just passing the payment details object down layers from component to all the way to the occ adapter

Reply:
After a discussion, we decided not to do it.